cardano_node_tests/tests/test_cli.py

```diff
@@ -290,6 +290,65 @@ def test_address_info_with_invalid_address(self, cluster: clusterlib.ClusterLib,
         assert "Invalid address" in err_str, err_str
 
 
+@pytest.mark.smoke
+class TestAddressKeyHash:
+    """Tests for cardano-cli address key-hash."""
+
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.parametrize("option", ("vkey", "vkey_file"))
+    def test_valid_verification_key(self, cluster: clusterlib.ClusterLib, option: str):
+        """Check `address key-hash` with valid verification key."""
+        temp_template = f"{common.get_test_id(cluster)}_{option}"
+
+        vkey_file = cluster.g_address.gen_payment_addr_and_keys(
+            name=temp_template,
+        ).vkey_file
+
+        if option == "vkey":
+            with open(vkey_file, encoding="utf-8") as infile:
+                # Ignore the first 4 chars, just an informative keyword
+                vkey = helpers.encode_bech32(
+                    prefix="vrf_vk", data=json.loads(infile.read().strip()).get("cborHex", "")[4:]
+                )
+
+        cluster.g_address.get_payment_vkey_hash(
+            payment_vkey=vkey if option == "vkey" else None,
+            payment_vkey_file=vkey_file if option == "vkey_file" else None,
+        )
```

Comment:
Let's add a "golden" vkey into `cardano_node_tests/tests/data` and check the expected hash value.